osl/interface_linux.go

```diff
@@ -241,8 +241,9 @@ func (n *networkNamespace) AddInterface(srcName, dstPrefix string, options ...If
 	if n.isDefault {
 		i.dstName = i.srcName
 	} else {
-		i.dstName = fmt.Sprintf("%s%d", i.dstName, n.nextIfIndex)
-		n.nextIfIndex++
+		var dstPrefix = i.dstName
```

Comment:
Fixed - using dstNamePrefix now. dstPrefix is for the name inside the host - thus using it here is wrong and confusing.